cloudwatch-papertrail.ts

```diff
@@ -46,9 +46,29 @@ function getEnvVarOrFail(varName: string): string {
   return value
 }
 
+// Should match for example: "[error] The database has exploded"
+const logLevelRegex = /^\[(\w+)\]/
+
+export function parseLogLevel(tsvMessage: string): string | undefined {
+  // Messages logged manually are tab separated value strings of three columns:
+  // date string (ISO8601), request ID, log message
+  const messageColumns = tsvMessage.split("\t")
+  if (messageColumns.length != 3) {
+    return undefined
+  }
+  const message = messageColumns[2]
+  const match = logLevelRegex.exec(message)
```

Comment:
We can combine both the split to date, request, message and log level parsing to single regex:
`/^.+\t.+\t\[(\w+)\]/` or something like this...

Reply:
Perhaps make it stricter than above...

Reply:
Fixed